src/index.html

```diff
@@ -5,10 +5,122 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>John Roberts' CV</title>
-    <link rel="stylesheet" href="./css/index.scss" />
-  </head>
-  <body>
+    <link rel="stylesheet" href="./css/index.scss"/>
+
+    <body>
     <div id="main"></div>
     <script src="./js/index.js"></script>
+      <nav> HOME </nav>
+      <button> Download my resume </button>
+
+      <!-- 主体第一部分 -->  
+      <h1>John Roberts's CV</h1>
+        <h2>BUSINESS EXPERT</h2>
+        <img src="smiley.gif" alt="Smiley face" height="42" width="42">
```

Comment:
`<img src="" alt="" width="" height="" />` img tag needs to be self-closed.
Good use of `alt, width, height` though.